charts/galoy/Chart.yaml

```diff
@@ -31,6 +31,11 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     version: 11.9.13
     condition: postgresql.enabled
+  - alias: kratospg
```

Comment:
should be kratos-pg imo

Reply:
yes, it was a problem in the templates/galoy-secrets.yaml

anyway the the kratos chart has an automigration init container which runs before the postgresql deployment in the same chatr so need to move the postgres instance out